### PR TITLE
required and minLength confuses schema validation

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#4392

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

File: src/registry_schemas/schemas/registrars_notation.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "properties": {
         "fileNumber": {
             "type": "string",
-            "minLength": 5,
+            "minLength": 0,
```
Comment (vysakh-menon-aot) [inline src/registry_schemas/schemas/registrars_notation.json:10]: Schema validates fileNumber as mandatory when we set minLength

File: src/registry_schemas/schemas/registrars_notation.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
         },
         "orderDetails": {
             "type": "string",
-            "minLength": 0,
+            "minLength": 1,
```
Comment (vysakh-menon-aot) [inline src/registry_schemas/schemas/registrars_notation.json:28]: Schema skip validate orderDetails as minLength as 0

File: src/registry_schemas/schemas/registrars_order.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "properties": {
         "fileNumber": {
             "type": "string",
-            "minLength": 5,
+            "minLength": 0,
```
Comment (vysakh-menon-aot) [inline src/registry_schemas/schemas/registrars_order.json:10]: Schema validates fileNumber as mandatory when we set minLength

File: src/registry_schemas/schemas/registrars_order.json [hunk 4]
```diff
@@ -25,7 +25,7 @@
         },
         "orderDetails": {
             "type": "string",
-            "minLength": 0,
+            "minLength": 1,
```
Comment (vysakh-menon-aot) [inline src/registry_schemas/schemas/registrars_order.json:28]: Schema skip validate orderDetails as minLength as 0

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**